### ISSUE TITLE
[Accessibility] Tooltip on hover only — doesn't work on touch devices

### ISSUE THREAD

**Issue by sshdopey** [Iris-IV/ProofOfHeart-frontend#1154]
## Problem
Fee explanation tooltip unreachable on mobile.

**Comment by Xhaka00** [comment]
@Xhaka00 has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> I would appreciate the chance to work on this. It’s the kind of task I’m looking to contribute to, and I’m ready to take responsibility for it and see it through to completion.


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/fd02cfec-3a14-41c9-8422-1fc99d8c7817) or assign @Xhaka00 to this issue.

**Comment by emirykl** [comment]
@emirykl has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> Hey, I want to take that issue


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/fd02cfec-3a14-41c9-8422-1fc99d8c7817) or assign @emirykl to this issue.

**Comment by Ajibose** [comment]
@Ajibose has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> I would love to work on this


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/fd02cfec-3a14-41c9-8422-1fc99d8c7817) or assign @Ajibose to this issue.

**Comment by 0xElyte** [comment]
@0xElyte has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> I'd love to resolve this fast!


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/fd02cfec-3a14-41c9-8422-1fc99d8c7817) or assign @0xElyte to this issue.
